pytest.ini

```diff
@@ -7,4 +7,4 @@ flakes-ignore =
 filterwarnings =
     ignore::DeprecationWarning
     ignore::PendingDeprecationWarning
-timeout = 100
+timeout = 300
```

Comment:
On purpose ?

Reply:
You betcha. The linting of the generated workflow can take quite some time occasionally. Was at 200 for part of the PR, but think it was just barely over that.